Added Object Properties

tawahle opened the pull request:
Da es an einigen deutschen Einrichtungen das Interesse besteht, die Projekte mit den institutionellen Projektpartnern zu verknüpfen, schlagen wir folgende Object Properties vor: vivo-de:kooperationspartner und
vivo-de:beteiligtAn, um die Abbildung von Kooperationspartnern der
Projekte zu ermöglichen.

tawahle committed:
Added Object Properties

Hinzugefügt Object Properties vivo-de:kooperationspartner und
vivo-de:beteiligtAn, um die Abbildung von Kooperationspartnern der
Projekte zu ermöglichen.

sonschul commented:
Bei den englischen Labels würde ich vielleicht noch als Änderung vorschlagen: 
 "engaged in" statt "involved in" 
 "collaborating organization" statt "collaborator" (eher Personen?)

tawahle commented:
> Bei den englischen Labels würde ich vielleicht noch als Änderung vorschlagen:
> "engaged in" statt "involved in"
> "collaborating organization" statt "collaborator" (eher Personen?)

Gute Idee! Ich werde die Änderungen einpflegen.

tawahle committed:
Änderung der englischen Bezeichnungen

Changes to the Englisch labels of vivo-de:beteiligtAn und
vivo-de:kooperationspartner

hauschke commented:
Vielen Dank für das Feedback. Ich veranlasse nun das Mergen.
